src/parser/monk/windwalker/modules/spells/azeritetraits/FuryOfXuen.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import Events from 'parser/core/Events';
+import Analyzer, { SELECTED_PLAYER_PET, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import AbilityTracker from 'parser/shared/modules/AbilityTracker';
+import {calculateAzeriteEffects} from 'common/stats';
+import SPELLS from 'common/SPELLS';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import { formatNumber, formatPercentage } from 'common/format';
+
+const furyOfXuenStats = traits => Object.values(traits).reduce((obj, rank) => {
+  const [haste] = calculateAzeriteEffects(SPELLS.FURY_OF_XUEN.id, rank);
+  obj.haste += haste;
+  return obj;
+}, {
+  haste: 0,
+});
+
+export const STAT_TRACKER = {
```

Comment:
PR #2791 changed the way stat buffs are added into the StatTracker. See the comment in the constructor of FuryOfXuen